frontend/src/audios.html

```diff
@@ -8,8 +8,17 @@
   - topnav
 jsFile: audios.js
 ---
-<div class="mt-4 flex">
-  <div id="filter-panel" class="fixed top-0 z-20 hidden h-full w-64 max-w-full flex-col bg-white text-sm open:flex md:static md:z-auto md:flex md:w-1/5 md:max-w-sm">
+<div
+  x-data="{open: false}"
+  x-init="$watch('open', (value) => $dispatch('toggle-backdrop', value))"
+  class="mt-4 flex"
+>
+  <div
+    x-cloak
+    @keydown.escape="open = false"
+    :data-open="open"
+    class="fixed top-0 z-20 hidden h-full w-64 max-w-full bg-white text-sm transition-transform data-[open]:flex data-[open]:flex-col md:static md:z-auto md:flex md:w-1/5 md:max-w-sm"
```

Comment:
In this case we don't need the data attributes. You can toggle specific classes via Alpine directly. See https://alpinejs.dev/directives/bind#binding-classes.

Reply:
binding classes from Alpine directly, data attribute removed.